test_data/macOS/clean/ls.json

```diff
@@ -17,6 +17,18 @@
                 "rpath"
             ],
             "Behaviors": [
+                {
```

Comment:
There seems to be some ordering oscillation going on here and it changes from run to run.